### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by brandonhundt**
- Much Improved a11y lighthouse score (compare to live site)
- Add webp to all picture presets to help serve a lighter page
- ran all the base images through an optimizer, saves a bit of payload!

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -63,6 +63,9 @@ compress_html:
   # ignore:
   #   envs: all
 
+sass:
+  style: compressed
```
Comment (brandonhundt) [inline _config.yml:67]: This was recommended to ensure SASS files are compressed.

File: _data/picture.yml [hunk 2]
```diff
@@ -20,11 +20,13 @@ presets:
     size: 100vw
 
   hero-card:
+    formats: [webp, original]
```
Comment (brandonhundt) [inline _data/picture.yml:23]: Images using the none default format weren't serving a webp, increasing load. Added here.

File: _includes/components/info-tiles.html [hunk 3]
```diff
@@ -19,11 +19,7 @@ <h2>{{ include.heading }}</h2>
     {%- capture attrs %}{{ attrs }} data-color="{{- info_tile.color -}}"{% endcapture -%}
   {%- endif -%}
   <div class="info-tile"{{ attrs }}>
-  {%- if include.heading -%}
-    <h4>{{ info_tile.heading }}</h4>
-  {%- else -%}
     <h3>{{ info_tile.heading }}</h3>
```
Comment (brandonhundt) [inline _includes/components/info-tiles.html:22]: The h4 wasn't in sequential order

File: _includes/navbar/navbar.html [hunk 4]
```diff
@@ -3,7 +3,7 @@
       <button popovertarget="navDrawerPopover" class="btn white">
         <span class="material-icons">menu</span>
       </button>
-      <a href="#" class="prx-logo">
+      <a href="#top" class="prx-logo" aria-label="Back to the top of the Annual Report">
```
Comment (brandonhundt) [inline _includes/navbar/navbar.html:6]: The link needed to be labeled.

File: _includes/page-types/playlist.html [hunk 5]
```diff
@@ -22,6 +22,7 @@ <h2>Listen to the 2024 PRX Annual Report podcast playlist</h2>
         height="593"
         width="100%"
         style="min-width: 300px;"
+        title="2024 PRX Annual Report Podcast Playlist"
```
Comment (brandonhundt) [inline _includes/page-types/playlist.html:25]: The iframe needed a label (maybe something to add to the defaults in Feeder??)

File: _includes/page-types/show.html [hunk 6]
```diff
@@ -33,7 +33,7 @@ <h2 class="show-title">{{ show.title }}</h2>
             </button>
             <div class="show-audio--controls">
               <span class="show-audio--slider">
-                <input type="range" defaultValue="0" min="0" max="1" step="0.01" />
+                <input id="progress-bar" type="range" defaultValue="0" min="0" max="1" step="0.01" aria-label="Player Progress" />
```
Comment (brandonhundt) [inline _includes/page-types/show.html:36]: Input needed a label, also added an id to further describe.

File: collections/_content/donor-list.html [hunk 7]
```diff
@@ -38,6 +38,8 @@ <h3 class="donor-list--level-label">
         <dt class="donor--name">{{ donor.label }}</dt>
         {%- if donor.location -%}
         <dd class="donor--location">{{ donor.location }}</dd>
+        {%- else -%}
+        <dd class="donor--none">Anonymous Location</dd>
```
Comment (brandonhundt) [inline collections/_content/donor-list.html:42]: Weird one, but when there's not a location included, was getting an a11y error for improper use of a dt. So I added in this bit but hid it via CSS. Its extra labelling.

File: _includes/components/info-tiles.html (hunk 3 above)
Comment (rpeterman-gp) [reply _includes/components/info-tiles.html:22]: Does this always being an h3 make a11y complain if the include didn't have a heading to create the preceding h2?

File: _data/picture.yml (hunk 2 above)
Comment (rpeterman-gp) [reply _data/picture.yml:23]: Good call.

File: _includes/components/info-tiles.html (hunk 3 above)
Comment (brandonhundt) [reply _includes/components/info-tiles.html:22]: It complains when you jump a level. In this case it jumped from h2 to h4, skipping h3

**Review by rpeterman-gp — APPROVED** [review]
:shipit:

**Pull request merged**